LuaTemplater.h

```diff
@@ -25,8 +25,5 @@ class LuaTemplater {
   void set(std::string const &name, std::vector<int> &values);
   void set(std::string const &name, std::vector<float> &values);
   std::string render(std::string const &template_string);
-
-private:
```

Comment:
I removed the function because all it did was copying a static string to a std::string and copying it again, just to use c_str() on it to access again. So I replaced the function call with accessing a static variable (and the variable is defined in LuaCppTemplater.cpp).